packages/sdk/src/collections/count.ts

```diff
@@ -0,0 +1,25 @@
+import type { Where } from 'payload'
+
+import type { PayloadSDK } from '../index.js'
+import type { CollectionSlug, PayloadGeneratedTypes, TypedLocale } from '../types.js'
+
+export type CountOptions<T extends PayloadGeneratedTypes, TSlug extends CollectionSlug<T>> = {
+  collection: TSlug
+  locale?: 'all' | TypedLocale<T>
+  where?: Where
+}
+
+export async function count<T extends PayloadGeneratedTypes, TSlug extends CollectionSlug<T>>(
+  sdk: PayloadSDK<T>,
+  options: CountOptions<T, TSlug>,
+  init?: RequestInit,
+): Promise<{ totalDocs: number }> {
```

Comment:
In these simple cases it may be convenient to define it from the core types so as not to repeat them: `ReturnType<Payload['count']>`.

In cases where there are type parameters like `CountOptions` it's probably not worth it though.